bin/console

```diff
@@ -10,6 +10,15 @@ if (!is_file(dirname(__DIR__).'/vendor/autoload_runtime.php')) {
 
 require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
 
+/**
+ * This file is a Symfony Front Controller (entrypoint)
```

Comment:
Just asking: is it correct to use the term "front controller" when we are not in the HTTP/web context? Thanks.

Reply:
Well this is a good question I can not answer
Entrypoint is ok, perhaps we could just write this

the aim or this PR is to show the 2 way or booting symfony

Reply:
@javiereguiluz hello

in this page: https://symfony.com/doc/current/the-fast-track/en/3-zero.html#initializing-the-project

`The bin/ directory contains the main CLI entry point: console. You will use it all the time.`

so what do you think I rewrite: `This file is a Symfony CLI entrypoint`